kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/pom.xml

```diff
@@ -156,6 +157,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+
+    <dependency>
```

Comment:
Is there any reason for declaring this dependency here? I see that it is already used in local-test-run profile.

Reply:
+1 good catch, let me check

Reply:
@sutaakar fixed, thank you